templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+  <li><a href="{% url 'taxi:index' %}"></a>Home page</li>
```

Comment:
The text 'Home page' should be placed inside the `<a>` tag to ensure it is clickable. It should be `<a href="{% url 'taxi:index' %}">Home page</a>`.